python/lsst/meas/extensions/gaap/_gaap.py

```diff
@@ -243,6 +245,37 @@ def getExecutionOrder(cls) -> float:
         # Docstring inherited.
         return cls.FLUX_ORDER
 
+    def _computeACF(self, kernel: lsst.afw.math.Kernel) -> lsst.afw.image.Image:
+        kernelImage = afwImage.ImageD(kernel.getDimensions())
+        kernel.computeImage(kernelImage, False)
+        acfArray = scipy.signal.correlate2d(kernelImage.array, kernelImage.array, boundary='fill')
+        acfImage = afwImage.ImageD(acfArray)
+        return acfImage
+
+    def _getFluxErrScaling(self, kernelACF: lsst.afw.image.Image,
+                           aperShape: lsst.afw.geom.Quadrupole) -> float:
+        """Returns the value by which the standard error has to be scaled due
+           to noise correlations.
+
+        Parameters
+        ----------
+        kernelACF : `lsst.afw.image.Image`
+            The auto-correlation function (ACF) of the PSF matching kernel.
+        aperShape : `lsst.afw.geom.Quadrupole`
+            The shape parameter of the Gaussian function which was used to
+            measure GAaP flux.
+
+        Returns
+        -------
+        fluxErrScaling : `float`
+            The factor by which the standard error on GAaP flux must be scaled.
+        """
+        aperShape2 = afwGeom.Quadrupole(2*aperShape.getParameterVector())
```

Comment:
Minor quibble, this should be named something like `aperShapeX2` because `aperShape2` would imply squared (as in `aperSigma2` below).

Reply:
Fair enough. I changed this variable name to `aperShapeX2` and merged.